src/main/java/com/frostcraft/admintools/VerifyCommand.java

```diff
@@ -0,0 +1,53 @@
+package com.frostcraft.admintools;
+
+import org.bukkit.Bukkit;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+public class VerifyCommand implements CommandExecutor {
+    Main plugin;
+
+    public VerifyCommand(Main plugin) { this.plugin = plugin; }
+
+    @Override
+    public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
+        Player player;
+        java.util.UUID uuid;
+        try {
+            player = (Player) sender;
+            uuid = player.getUniqueId();
+        }
+        catch (Exception e) {
+            sender.sendMessage("This command must be issued by a player.");
+            return false;
+        }
+
+        // Make sure min args are met.
+        // TODO: Find a better way to do this.
```

Comment:
Noticed the comment looking for a better way - what about checking for the exact expected length rather than less than 1?